ui/src/pages/version/components/forms/validation/schema.js

```diff
@@ -67,6 +67,17 @@ const environmentVariableSchema = yup.object().shape({
   value: yup.string(),
 });
 
+const stringValueSchema = (name) => yup.mixed().test(
```

Comment:
`yup.mixed()` will take in any type of variable, therefore both the `object` and `string` will pass this validation check. the further validation will be put in the `.test` part, where it will compare the original schema to the input or the value of `jsonPath` (depending on input type)

Reply:
Since `jsonPath` is hardcoded in this function, could we either:
1) Rename the name of this function to reflect that it only checks for the value of the string that corresponds to the `jsonPath` key, e.g. maybe something like `jsonPathValueSchema`, or 
2) Make this function take in the actual key that it needs to access the string value i.e. make it the key value an argument in this function alongside `name`

Reply:
Let me change it to point 2 and test for a bit